pkg/gadgets/snisnoop/tracer/tracer.go

```diff
@@ -33,6 +32,8 @@ import (
 	eventtypes "github.com/kinvolk/inspektor-gadget/pkg/types"
 )
 
+//go:generate sh -c "echo $CLANG_OS_FLAGS; GOOS=$(go env GOHOSTOS) GOARCH=$(go env GOHOSTARCH) go run github.com/cilium/ebpf/cmd/bpf2go -target bpfel -cc clang snisnoop ./bpf/snisnoop.c -- $CLANG_OS_FLAGS -I./bpf/ -I../../.. -target bpf -D__TARGET_ARCH_x86"
```

Comment:
Is `-I../../..` needed for something? Since snisnoop and dns are network programs, they are not dependent on a specific kernel, they don't include vmlinux.h so it might not be necessary.